internal/devconfig/configfile/env.go

```diff
@@ -1,6 +1,63 @@
 package configfile
 
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
 func (c *ConfigFile) IsEnvsecEnabled() bool {
-	// envsec for legacy.
-	return c.EnvFrom == "envsec" || c.EnvFrom == "jetpack-cloud"
+	// envsec for legacy. jetpack-cloud for legacy
+	return c.EnvFrom == "envsec" || c.EnvFrom == "jetpack-cloud" || c.EnvFrom == "jetify-cloud"
+}
+
+func (c *ConfigFile) IsdotEnvEnabled() bool {
+	// filename has to end with .env
+	return filepath.Ext(c.EnvFrom) == ".env"
+}
+
+func (c *ConfigFile) ParseEnvsFromDotEnv() (map[string]string, error) {
+	// This check should never happen because we call IsdotEnvEnabled
+	// before calling this method. But having it makes it more robust
+	// in case if anyone uses this method without the IsdotEnvEnabled
+	if !c.IsdotEnvEnabled() {
+		return nil, fmt.Errorf("env file does not have a .env extension")
+	}
+
+	file, err := os.Open(c.EnvFrom)
```

Comment:
This will fail if devbox is call from any other directory that is not where the devbox.json is. I don't think it makes sense to make this relative from the working directory, it should be relative from the config file directory.

Reply:
I agree with this, my project failed due to it.

Reply:
@varunpalekar thanks for reminding on the issue. I made a [PR](https://github.com/jetify-com/devbox/pull/2355) to fix this, once merged it will be available in the next version of devbox.